substrate/primitives/version/proc-macro/src/decl_runtime_version.rs

```diff
@@ -125,7 +130,10 @@ impl ParseRuntimeVersion {
 		} else if field_name == "transaction_version" {
 			parse_once(&mut self.transaction_version, field_value, Self::parse_num_literal)?;
 		} else if field_name == "state_version" {
-			parse_once(&mut self.state_version, field_value, Self::parse_num_literal_u8)?;
+			log::warn!("{STATE_VERSION_DEPRECATION_WARN}");
```

Comment:
https://github.com/ggwpez/proc-macro-warning please use this here.

Reply:
@bkchr I have updated to use the `proc-macro-warning` crate until now. I derived the change from the examples. Let me know if something can be improved there